src/resources/utils.py

```
@@ -367,32 +372,45 @@ def clean_up_the_rest(database: Database, namespace: str = 'default') -> str:
         database: Database wrapper used to look up the known analysis ids.
+        hub_client: FLAME Hub client.
         namespace: Namespace to search in.
 
     Returns:
         A human-readable newline-separated summary counting the zombies
         deleted per resource type.
     """
     known_analysis_ids = database.get_analysis_ids()
+    if hub_client is not None:
+        validated_analysis_ids = _validate_analyses_with_hub(known_analysis_ids, hub_client)
+        for id in known_analysis_ids:
```

Ruff (0.15.12)

[error] Variable `id` is shadowing a Python builtin

(A001)